src/main/kotlin/org/jitsi/rtp/rtcp/RtcpPacket.kt

```diff
@@ -71,6 +71,7 @@ abstract class RtcpPacket(
     abstract override fun clone(): RtcpPacket
 
     companion object {
+        //TODO we need to have a limit
```

Comment:
we don't need a limit here because of the length field, no?

Reply:
That's my point -- there's no 'length' passed to parse()

Reply:
right but there's a length field in the rtcp header

Reply:
Yes, but that comes from the network

Reply:
i.e. it could point further than the buffer's length

Reply:
ah i see what you mean, yeah ok.